crates/burn-router/src/runner.rs

```diff
@@ -1202,6 +1202,9 @@ where
                     handles.register_float_tensor::<B>(&desc.out.id, output);
                 }
             },
+            OperationDescription::Custom(_) => {
+                panic!("Can't execute custom operation here")
```

Comment:
At some point we should resolve a function based on the custom description id and call it with the handles and the description as input.